docs/appendices/release-notes/5.4.6.rst

```diff
@@ -46,8 +46,8 @@ See the :ref:`version_5.4.0` release notes for a full list of changes in the
 Fixes
 =====
 
-- Fixed an issue that caused queries with a ``NOT`` expression in the where
-  clause to fail on evaluation ``NULL`` correctly.
+- Fixed an issue that caused queries with a ``NOT`` expression in the
+  where-clause to fail evaluating ``NULL`` correctly.
```

Comment:
I think in general we use:
```suggestion
  ``WHERE`` clause to fail evaluating ``NULL`` correctly.
```
Maybe you can change this accordingly also here: https://github.com/crate/crate/pull/15123/files#diff-099841c0a4419b13dca629303a4fb7fc73f350df360f9bb7db73f48a6828549aR68